### ISSUE TITLE
read "Development of a Funding Mechanism for Sustaining Open Source Software for European Public Services"

### ISSUE THREAD

**Issue by chadwhitacre** [chadwhitacre/openpath#35]
[**Development of a Funding Mechanism for Sustaining Open Source Software for European Public Services**](https://joinup.ec.europa.eu/sites/default/files/news/2022-04/Development%20of%20a%20Funding%20Mechanism%20for%20Sustaining%20Open%20Source%20Software%20for%20European%20Public%20Services.pdf) (PDF)

[h/t](https://github.com/KevinCrosby); [blog announcement](https://joinup.ec.europa.eu/collection/fosseps/news/funding-sustainability)

**Comment by chadwhitacre** [comment]
The study asserts "a clear need for a European funding mechanism to help sustain
critical open source software communities" (7), and describes a proposal for such a fund, which they stop short of _recommending_ (7; seems like semantics to me). The proposed fund seems to me roughly equivalent to [STF](https://www.sovereigntechfund.de/) but scoped to EU vs. DE. Its mission (44):

> The European funding mechanism will enable European public services to use, safely rely on, encourage innovation in and contribute to the long term sustainability of open source software and its associated ecosystems within Europe, and support open source projects that contribute to a long term positive societal impact.

The report focuses throughout on "projects deemed critical" and "individual developers/small developer groups" (6, 8, 30, 43). I like this part. 😎

The fund is sketched as €15 million over 3 years, supporting up to 300 projects per year with payments from €5k to €50k (48) with 12% administrative overhead (49). The fund is a new, "independent entity" reliant on "public financing" (37). Funds are awarded through an application process for a specific scope of work (50). Awards are paid out over a period ranging from six months to two years, with a larger initial payment followed by subsequent payments based on successful completion of the work (52-53).

There's a marked resentment and suspicion of "US dominance" (58). They have a particularly negative view of Microsoft's acquisition of GitHub (31, 57), to the extent that they exclude GitHub Sponsors from their analysis (27)! They also exclude Liberapay. 🥹 🫶

**tl;dr** for me here is that this is a potential EU parallel to what I envision for Software Commons, some similarities, more differences. I envision funds coming from the private sector and flowing through foundations to devs based on past year's impact, vs. from the public sector directly to small devs/projects based on proposed future work. Likely worth connecting with [Saranjit](https://www.linkedin.com/in/saranjitarora/) at some point to compare notes on complementary approaches.
